index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css"
+    integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous" />
+  <link rel="stylesheet" href="style.css">
+  <title>Yuriy's Portfolio</title>
+</head>
+
+<body>
+  <div class="container">
+    <header>
+      <span class="logo"><a class="logo-text" title="logo" href="#">My Logo</a></span>
+      <nav>
+        <i class="fas fa-bars hidden"></i>
+        <ul class="menu">
+          <li class="menu-item"><a href="#" title="Portfolio">Portfolio</a></li>
+          <li class="menu-item"><a href="#" title="About">About</a></li>
+          <li class="menu-item"><a href="#" title="Contact">Contact</a></li>
+        </ul>
+      </nav>
+    </header>
+    <section class="headline">
```

Comment:
Background images do not show according to the design on Mobile view large. Please make sure that your page looks the same on all mobile views.
Design:
![orignal](https://user-images.githubusercontent.com/35504149/138300637-7f49c5c6-426a-47bb-85f4-93efa7765d44.png)
Yours
![yours](https://user-images.githubusercontent.com/35504149/138300688-da4ae20f-743b-4da3-ba39-5f47265e485e.png)
: